### DISCUSSION TITLE
Update to German locale [de]

### DISCUSSION THREAD

**Discussion by calipo69** [aclap-dev/video-downloadhelper#667] [Language & Translation]
Hello!
I went through the whole **German translation**, put there the missing parts and changed all files from "2nd person plural" to "2nd person singular". In my opinion it is much better.
Hope I could have helped.
[de.txt](https://github.com/aclap-dev/video-downloadhelper/files/15445623/de.txt)

Calipo69 from Vienna

**Comment by paulrouget** [comment]
Thank you so much ❤️  I'll add that to the next release :)
